server/src/main/java/io/seata/server/storage/redis/lock/RedisLocker.java

```diff
@@ -268,7 +272,14 @@ private boolean acquireLockByLua(Jedis jedis, List<RowLock> rowLocks) {
         args.add(lockKeysString.toString());
         // reset args index 2
         args.set(1, String.valueOf(args.size()));
-        String xIdOwnLock = (String) jedis.evalsha(ACQUIRE_LOCK_SHA, keys, args);
+        String xIdOwnLock;
+        try {
+            xIdOwnLock = (String) jedis.evalsha(ACQUIRE_LOCK_SHA, keys, args);
+        } catch (JedisNoScriptException e) {
+            LOGGER.warn("jedis ex: " + e.getMessage());
```

Comment:
To highlight that the exception has been reloaded lua scripts

Reply:
done